### ISSUE TITLE
Fetched file: netgen-5.0.0.tar.gz VERIFY FAILED!

### ISSUE THREAD

**Issue by apieum** [gentoo-science/sci#407]
Hi, I can't install sci-mathematics/netgen-5.0.0, due to a bad file size.
emerge  tried differents sources but each reports:
```
!!! Fetched file: netgen-5.0.0.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      1835004
!!! Expected: 1835590
Refetching... File renamed to '/usr/portage/distfiles/netgen-5.0.0.tar.gz._checksum_failure_.u0gqvc'
```
Each downloaded file is 1835004, I guess error is about expected size.

build.log
```
>>> Downloading 'http://distfiles.gentoo.org/distfiles/netgen-5.0.0.tar.gz'
--2015-04-25 22:41:09--  http://distfiles.gentoo.org/distfiles/netgen-5.0.0.tar.gz
Resolving distfiles.gentoo.org... 137.226.34.42, 216.165.129.135, 140.211.166.134, ...
Connecting to distfiles.gentoo.org|137.226.34.42|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-04-25 22:41:09 ERROR 404: Not Found.

>>> Downloading 'http://freefr.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz'
--2015-04-25 22:41:09--  http://freefr.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz
Resolving freefr.dl.sourceforge.net... 2a01:e0d:1:8:58bf:fa88:0:1, 88.191.250.136
Connecting to freefr.dl.sourceforge.net|2a01:e0d:1:8:58bf:fa88:0:1|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1835004 (1,7M) [application/x-gzip]
Saving to: ‘/usr/portage/distfiles/netgen-5.0.0.tar.gz’


/usr/portage/distfi   0%[                      ]       0  --.-KB/s             
usr/portage/distfil   3%[                      ]  70,12K   349KB/s             
sr/portage/distfile  12%[=>                    ] 227,75K   496KB/s             
r/portage/distfiles  25%[====>                 ] 460,88K   698KB/s             
/portage/distfiles/  36%[=======>              ] 653,38K   760KB/s             
portage/distfiles/n  47%[=========>            ] 845,88K   798KB/s             
ortage/distfiles/ne  57%[===========>          ]   1,01M   824KB/s             
rtage/distfiles/net  68%[==============>       ]   1,20M   841KB/s             
tage/distfiles/netg  79%[================>     ]   1,39M   857KB/s             
age/distfiles/netge  90%[==================>   ]   1,58M   868KB/s             
/usr/portage/distfi 100%[=====================>]   1,75M   876KB/s   in 2,0s   

2015-04-25 22:41:11 (876 KB/s) - ‘/usr/portage/distfiles/netgen-5.0.0.tar.gz’ saved [1835004/1835004]

!!! Fetched file: netgen-5.0.0.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      1835004
!!! Expected: 1835590
Refetching... File renamed to '/usr/portage/distfiles/netgen-5.0.0.tar.gz._checksum_failure_.u0gqvc'

>>> Downloading 'http://sunet.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz'
--2015-04-25 22:41:11--  http://sunet.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz
Resolving sunet.dl.sourceforge.net... 194.71.11.73
Connecting to sunet.dl.sourceforge.net|194.71.11.73|:80... connected.
HTTP request sent, awaiting response... 302 Found
Location: http://downloads.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz?download&failedmirror=sunet.dl.sourceforge.net [following]
--2015-04-25 22:41:11--  http://downloads.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz?download&failedmirror=sunet.dl.sourceforge.net
Resolving downloads.sourceforge.net... 216.34.181.59
Connecting to downloads.sourceforge.net|216.34.181.59|:80... connected.
HTTP request sent, awaiting response... 302 Found
Location: http://freefr.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz [following]
--2015-04-25 22:41:12--  http://freefr.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz
Resolving freefr.dl.sourceforge.net... 2a01:e0d:1:8:58bf:fa88:0:1, 88.191.250.136
Connecting to freefr.dl.sourceforge.net|2a01:e0d:1:8:58bf:fa88:0:1|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1835004 (1,7M) [application/x-gzip]
Saving to: ‘/usr/portage/distfiles/netgen-5.0.0.tar.gz’


/usr/portage/distfi   0%[                      ]       0  --.-KB/s             
usr/portage/distfil   3%[                      ]  58,86K   289KB/s             
sr/portage/distfile  13%[=>                    ] 241,50K   597KB/s             
r/portage/distfiles  24%[====>                 ] 434,00K   718KB/s             
/portage/distfiles/  34%[======>               ] 626,50K   778KB/s             
portage/distfiles/n  45%[=========>            ] 814,62K   810KB/s             
ortage/distfiles/ne  56%[===========>          ]   1009K   835KB/s             
rtage/distfiles/net  67%[=============>        ]   1,18M   852KB/s             
tage/distfiles/netg  78%[================>     ]   1,37M   866KB/s             
age/distfiles/netge  89%[==================>   ]   1,56M   875KB/s             
/usr/portage/distfi 100%[=====================>]   1,75M   884KB/s   in 2,0s   

2015-04-25 22:41:14 (884 KB/s) - ‘/usr/portage/distfiles/netgen-5.0.0.tar.gz’ saved [1835004/1835004]

!!! Fetched file: netgen-5.0.0.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      1835004
!!! Expected: 1835590
Refetching... File renamed to '/usr/portage/distfiles/netgen-5.0.0.tar.gz._checksum_failure_.u0gqvc'

>>> Downloading 'http://ignum.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz'
--2015-04-25 22:41:14--  http://ignum.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz
Resolving ignum.dl.sourceforge.net... 2001:1ab0:7e1f:1:230:48ff:fed1:9c0a, 62.109.128.11
Connecting to ignum.dl.sourceforge.net|2001:1ab0:7e1f:1:230:48ff:fed1:9c0a|:80... failed: No route to host.
Connecting to ignum.dl.sourceforge.net|62.109.128.11|:80... failed: Connection timed out.
Retrying.

--2015-04-25 22:42:18--  (try: 2)  http://ignum.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz
Connecting to ignum.dl.sourceforge.net|2001:1ab0:7e1f:1:230:48ff:fed1:9c0a|:80... failed: No route to host.
Connecting to ignum.dl.sourceforge.net|62.109.128.11|:80... failed: Connection timed out.
Retrying.

--2015-04-25 22:43:23--  (try: 3)  http://ignum.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz
Connecting to ignum.dl.sourceforge.net|2001:1ab0:7e1f:1:230:48ff:fed1:9c0a|:80... failed: No route to host.
Connecting to ignum.dl.sourceforge.net|62.109.128.11|:80... failed: Connection timed out.
Giving up.

>>> Downloading 'http://nchc.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz'
--2015-04-25 22:44:26--  http://nchc.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz
Resolving nchc.dl.sourceforge.net... 2001:e10:ffff:1f02::17, 211.79.60.17
Connecting to nchc.dl.sourceforge.net|2001:e10:ffff:1f02::17|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1835004 (1,7M) [application/x-gzip]
Saving to: ‘/usr/portage/distfiles/netgen-5.0.0.tar.gz’


/usr/portage/distfi   0%[                      ]       0  --.-KB/s             
usr/portage/distfil   0%[                      ]  16,24K  46,5KB/s             
sr/portage/distfile   2%[                      ]  46,49K  66,0KB/s             
r/portage/distfiles   5%[>                     ]  97,36K  92,9KB/s             
/portage/distfiles/  11%[=>                    ] 209,86K   121KB/s             
portage/distfiles/n  21%[===>                  ] 380,61K   189KB/s             
ortage/distfiles/ne  32%[======>               ] 589,61K   266KB/s             
rtage/distfiles/net  35%[======>               ] 639,11K   263KB/s             
tage/distfiles/netg  42%[========>             ] 768,36K   284KB/s             
age/distfiles/netge  50%[==========>           ] 912,74K   314KB/s             
ge/distfiles/netgen  54%[==========>           ] 974,61K   314KB/s   eta 3s    
e/distfiles/netgen-  62%[============>         ]   1,09M   337KB/s   eta 3s    
/distfiles/netgen-5  66%[=============>        ]   1,16M   337KB/s   eta 3s    
distfiles/netgen-5.  73%[===============>      ]   1,28M   351KB/s   eta 3s    
istfiles/netgen-5.0  81%[================>     ]   1,42M   368KB/s   eta 3s    
stfiles/netgen-5.0.  86%[==================>   ]   1,51M   373KB/s   eta 1s    
tfiles/netgen-5.0.0  94%[===================>  ]   1,66M   390KB/s   eta 1s    
files/netgen-5.0.0.  99%[====================> ]   1,73M   390KB/s   eta 1s    
/usr/portage/distfi 100%[=====================>]   1,75M   390KB/s   in 4,6s   

2015-04-25 22:44:32 (390 KB/s) - ‘/usr/portage/distfiles/netgen-5.0.0.tar.gz’ saved [1835004/1835004]

!!! Fetched file: netgen-5.0.0.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      1835004
!!! Expected: 1835590
Refetching... File renamed to '/usr/portage/distfiles/netgen-5.0.0.tar.gz._checksum_failure_.u0gqvc'

>>> Downloading 'http://surfnet.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz'
--2015-04-25 22:44:32--  http://surfnet.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz
Resolving surfnet.dl.sourceforge.net... 2001:620:0:1b::21, 130.59.138.21
Connecting to surfnet.dl.sourceforge.net|2001:620:0:1b::21|:80... failed: No route to host.
Connecting to surfnet.dl.sourceforge.net|130.59.138.21|:80... failed: Connection timed out.
Retrying.

--2015-04-25 22:45:36--  (try: 2)  http://surfnet.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz
Connecting to surfnet.dl.sourceforge.net|2001:620:0:1b::21|:80... failed: No route to host.
Connecting to surfnet.dl.sourceforge.net|130.59.138.21|:80... failed: Connection timed out.
Retrying.

--2015-04-25 22:46:41--  (try: 3)  http://surfnet.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz
Connecting to surfnet.dl.sourceforge.net|2001:620:0:1b::21|:80... failed: No route to host.
Connecting to surfnet.dl.sourceforge.net|130.59.138.21|:80... failed: Connection timed out.
Giving up.

>>> Downloading 'http://heanet.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz'
--2015-04-25 22:47:44--  http://heanet.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz
Resolving heanet.dl.sourceforge.net... 2001:770:18:aa40::c101:c142, 193.1.193.66
Connecting to heanet.dl.sourceforge.net|2001:770:18:aa40::c101:c142|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1835004 (1,7M) [application/x-gzip]
Saving to: ‘/usr/portage/distfiles/netgen-5.0.0.tar.gz’


/usr/portage/distfi   0%[                      ]       0  --.-KB/s             
usr/portage/distfil   1%[                      ]  27,50K   134KB/s             
sr/portage/distfile   7%[>                     ] 132,00K   322KB/s             
r/portage/distfiles  13%[=>                    ] 242,00K   368KB/s             
/portage/distfiles/  31%[=====>                ] 561,00K   654KB/s             
portage/distfiles/n  42%[========>             ] 753,50K   712KB/s             
ortage/distfiles/ne  52%[==========>           ] 946,00K   752KB/s             
rtage/distfiles/net  63%[============>         ]   1,11M   781KB/s             
tage/distfiles/netg  74%[===============>      ]   1,30M   802KB/s             
age/distfiles/netge  83%[=================>    ]   1,46M   782KB/s             
ge/distfiles/netgen  98%[====================> ]   1,72M   837KB/s             
/usr/portage/distfi 100%[=====================>]   1,75M   838KB/s   in 2,1s   

2015-04-25 22:47:46 (838 KB/s) - ‘/usr/portage/distfiles/netgen-5.0.0.tar.gz’ saved [1835004/1835004]

!!! Fetched file: netgen-5.0.0.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      1835004
!!! Expected: 1835590
Refetching... File renamed to '/usr/portage/distfiles/netgen-5.0.0.tar.gz._checksum_failure_.u0gqvc'

>>> Downloading 'http://colocrossing.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz'
--2015-04-25 22:47:47--  http://colocrossing.dl.sourceforge.net/project/netgen-mesher/netgen-mesher/5.0/netgen-5.0.0.tar.gz
Resolving colocrossing.dl.sourceforge.net... 75.127.11.234
Connecting to colocrossing.dl.sourceforge.net|75.127.11.234|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1835004 (1,7M) [application/x-gzip]
Saving to: ‘/usr/portage/distfiles/netgen-5.0.0.tar.gz’


/usr/portage/distfi   0%[                      ]       0  --.-KB/s             
usr/portage/distfil   1%[                      ]  33,67K   109KB/s             
sr/portage/distfile   5%[>                     ] 101,55K   199KB/s             
r/portage/distfiles  11%[=>                    ] 209,02K   294KB/s             
/portage/distfiles/  16%[==>                   ] 291,03K   319KB/s             
portage/distfiles/n  20%[===>                  ] 368,80K   306KB/s             
ortage/distfiles/ne  40%[=======>              ] 720,91K   513KB/s             
rtage/distfiles/net  44%[========>             ] 793,02K   480KB/s             
tage/distfiles/netg  62%[============>         ]   1,09M   600KB/s             
age/distfiles/netge  71%[==============>       ]   1,24M   617KB/s             
ge/distfiles/netgen  80%[================>     ]   1,41M   636KB/s             
e/distfiles/netgen-  89%[==================>   ]   1,57M   649KB/s             
/distfiles/netgen-5  98%[====================> ]   1,73M   663KB/s             
/usr/portage/distfi 100%[=====================>]   1,75M   666KB/s   in 2,7s   

2015-04-25 22:47:50 (666 KB/s) - ‘/usr/portage/distfiles/netgen-5.0.0.tar.gz’ saved [1835004/1835004]

!!! Fetched file: netgen-5.0.0.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      1835004
!!! Expected: 1835590
Refetching... File renamed to '/usr/portage/distfiles/netgen-5.0.0.tar.gz._checksum_failure_.u0gqvc'

!!! Couldn't download 'netgen-5.0.0.tar.gz'. Aborting.
[31;01m * [39;49;00mFetch failed for 'sci-mathematics/netgen-5.0.0', Log file:
[31;01m * [39;49;00m '/var/tmp/portage/sci-mathematics/netgen-5.0.0/temp/build.log'

```

**Comment by marbre** [comment]
Fixed with commit 47ece0ee4df44e83e3501bcfa8799ed0030ed977.

**Comment by apieum** [comment]
Thanks !
